Cargo.toml

```diff
@@ -38,7 +39,7 @@ alloy-rlp = "0.3"
 # crypto
 elliptic-curve = { version = "0.13.5", default-features = false, features = ["std"] }
 generic-array = { version = "0.14.7", default-features = false, features = ["std"] }
-k256 = { version = "0.13.2", default-features = false, features = ["ecdsa", "std"] }
+k256 = { version = "0.13.2", default-features = false, features = ["ecdsa", "std", "pem"] }
```

Comment:
can avoid adding the feature to the workspace by specifying in the relevant crate's `Cargo.toml` as `k256 = { workspace = true, features = ["pem"] }`